packages/cli/src/utils.ts

```diff
@@ -227,30 +224,31 @@ export function pluralize(label: string, num: number) {
 
 export function handleError(e: Error, ref: string) {
   switch (e.constructor) {
+    case HandledError: {
+      throw e;
```

Comment:
What is that for? Did not get it) And why we do not wrap it in `exitWithError` ?

Reply:
When we face a HandledError instance we know that it has been already handled by someone else. So we simply re-throw the error forward.

Reply:
For other cases we want to exit with some pretty error message.